CMakeLists.txt

```diff
@@ -118,25 +118,25 @@ install(
   PATTERN "openmp_wrappers" EXCLUDE)
 
 if (UNIX)
+    if(POLICY CMP0037)
+        cmake_policy(PUSH)
+        cmake_policy(SET CMP0037 OLD)
+    endif()
```

Comment:
Why CMP0037 cmake policy is introduced once again?

Reply:
CMake 2.8.12 and lower allowed unrestricted choice for the target name. CMP0037 OLD cmake policy allows the use of restricted target names such as 'target'.

Reply:
We've already discussed it: hipify-clang has a restriction on cmake version (see above):
```cmake
cmake_minimum_required(VERSION 3.5.1)
```

Reply:
We don't have to use the lower cmake version. The policy allows the use of restcricted target names.

Reply:
So, you need `CMP0037 OLD` policy only to use `package` as a target name instead of `pkg_hipify-clang`, right?
Maybe better not to use a reserved word `package`?

Reply:
@searlmc1, Do we need exactly `package` as a target name?

Reply:
@searlmc1, @aakanksha555, any updates?